web/text_layer_builder.js

```diff
@@ -79,22 +81,23 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
      *   for specified amount of ms.
      */
     render: function TextLayerBuilder_render(timeout) {
-      if (!this.textContent || this.renderingDone) {
+      if (!this.textContentStream || this.renderingDone) {
         return;
       }
       this.cancel();
 
       this.textDivs = [];
       var textLayerFrag = document.createDocumentFragment();
       this.textLayerRenderTask = renderTextLayer({
-        textContent: this.textContent,
+        textContentStream: this.textContentStream,
```

Comment:
Unless we completely remove the ability to use `setTextContent`, I think that you need to have both of these here:
```javascript
  textContent: this.textContent,
  textContentStream: this.textContentStream,
```